### ISSUE TITLE
[BITV] 9.4.1.2/2.2 - The "Search" button that opens a search dialog should not be implemented as a link, but as a button. In addition, the aria-expanded attribute should be used to inform users whether the search area is open or closed. Similar issues also apply to the "Notifications" and "Search contacts" links. (9)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37094]
- [x] https://github.com/nextcloud-libraries/nextcloud-vue/pull/4481
- [x] Release
- [x] https://github.com/nextcloud/notifications/pull/1676
- [x] https://github.com/nextcloud/server/pull/40692

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v2-n2

</details>

- [x] 9.4.1.2/3.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n2
- [x] 9.4.1.2/4.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v4-n2
- [x] 9.4.1.2/5.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v5-n2
- [x] 9.4.1.2/6.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v6-n2
- [x] 9.4.1.2/7.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v7-n2
- [x] 9.4.1.2/8.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v8-n2
- [x] 9.4.1.2/9.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v9-n3
- [x] 9.4.1.2/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n2
